test/acceptance-tests/workspace-devtime/step-definitions/createModel.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+var async = require('async');
+var app = require('../../../../');
+var chai = require('chai');
+var workspaceManager = require('../../../../component/workspace-manager.js');
+var loopback = require('loopback');
+var path = require('path');
+var util = require('util');
+var ModelDefinition = app.models.ModelDefinition;
+app.on('booted', function() {
+  app.emit('ready');
+});
+
+module.exports = function() {
+  var testsuite = this;
+  this.Given(/^that I have loaded the workspace$/, function(next) {
+    next();
+  });
+
+  this.When(/^I create model '(.+)'$/, function(modelName, next) {
+    testsuite.modelId = 'common.' + modelName;
+    var model = {
+      'id': testsuite.modelId,
+      'name': modelName,
+      'readonly': true,
+      'plural': 'customers',
+      'strict': true,
+      'public': true,
+      'idInjection': true,
+    };
+    testsuite.modelName = modelName;
+    ModelDefinition.create(model, {}, function(err, data) {
+      if (err) return next(err);
+      testsuite.modelDef = model;
+      next();
+    });
+  });
+
+  this.Then(/^the model definition is created$/, function(next) {
+    var expect = chai.expect;
```

Comment:
move the chai helper out to test/helpers/expect.js, then `var expect = require('../../../helpers/expect');`